baselines/diabetic_retinopathy_detection/jax_finetune_deterministic.py

```diff
@@ -65,7 +70,8 @@ def main(argv):
   config = FLAGS.config
 
   # Wandb and Checkpointing Setup
-  wandb_run, output_dir = vit_utils.maybe_setup_wandb(config)
+  output_dir = FLAGS.output_dir
+  wandb_run, output_dir = vit_utils.maybe_setup_wandb(FLAGS)
```

Comment:
Bug here I forgot to fix. Didn't notice it as this line isn't run internally. Namely, the input should be `config`, not `FLAGS`.